packages/aws-cdk-lib/aws-stepfunctions-tasks/lib/glue/start-job-run.ts

```diff
@@ -69,6 +84,10 @@ export class GlueStartJobRun extends sfn.TaskStateBase {
 
     validatePatternSupported(this.integrationPattern, GlueStartJobRun.SUPPORTED_INTEGRATION_PATTERNS);
 
+    if ((!props.workerType && props.numberOfWorkers !== undefined) || (props.workerType && props.numberOfWorkers === undefined)) {
+      throw new Error('Both workerType and numberOfWorkers must be set');
+    }
```

Comment:
If we can't have one without the other maybe we can make separate interface with just these 2 properties and have both of them be required in that interface? I'll see if i can find an example of this elsewhere in the CDK

Reply:
I think your suggestion is better as well. 
I extracted the worker settings into a workerConfiguration.